cyber-security/modules/csls_s3_object_logging/iam.tf

```diff
@@ -0,0 +1,20 @@
+data "aws_iam_policy_document" "cloudwatch_event_logging_policy_doc" {
+  statement {
+    actions = [
+      "logs:CreateLogStream",
+      "logs:PutLogEvents",
+    ]
+
+    resources = ["arn:aws:logs:eu-west-2:*:log-group:/aws/events/*:*"]
```

Comment:
Is it possible to determine the destination event group name instead of `*`?

Reply:
Done